herbert_core/configuration/@config_store/config_store.m

```diff
@@ -31,13 +31,13 @@
             if is_virtual
                 newStore.config_folder_name_ = ['mprogs_config_',type];
             end
-            [is_virtual,build_name] = is_jenkins();
+            [is_virtual,build_name,workspace] = is_jenkins();
```

Comment:
Looking at the use of is_virtual, it appears to really mean - is running on an instance of the platform (`is_platform`) for the call that returned it. For clarity it may be easier to rename these as `is_on_idaaas`, or `is_on_jenkins`, etc

Reply:
I think this is purely to maintain syntactic compatibility with other `is*` functions: `isstruct`, etc. I don't think there's really any ambiguity here warranting a name change throughout.

Reply:
is this workspace/config fix ONLY required when running as part of a Jenkins job, or is this more general behaviour change?